software/contrib/cvecorder.py

```diff
@@ -100,29 +117,35 @@ def b1Pressed():
             for n in range (0, self.stepLength):
                 self.CVR[self.ActiveBank][self.ActiveCvr][n] = 0
 
-        # B2 Long press
-        @b2.handler_falling
-        def b2PressedLong():
-            # 2000ms press clears all banks
-            if ticks_diff(ticks_ms(), b2.last_pressed()) >  2000:
-                self.clearCvrs('all')
-                #self.bankToSave = self.ActiveBank
-                #self.saveState()
-                # reverse the ActiveCvr increment caused by the initial button press
-                if self.ActiveCvr > 0:
-                    self.ActiveCvr -= 1
-                else:
-                    self.ActiveCvr = self.numCVR
-            # 500ms press clears the active bank
-            elif ticks_diff(ticks_ms(), b2.last_pressed()) >  500:
-                self.clearCvrs(self.ActiveBank)
-                self.bankToSave = self.ActiveBank
-                self.saveState()
-                # reverse the ActiveCvr increment caused by the initial button press
-                if self.ActiveCvr > 0:
-                    self.ActiveCvr -= 1
-                else:
-                    self.ActiveCvr = self.numCVR
+        # # B2 Long press
```

Comment:
[optional] Consider just deleting this block instead of leaving it commented out. This PR exists as record of what was removed and is easily referenced by you or anyone attempting to fix it. It also has all the context of the issue in the PR description. If we get in the habit of leaving commented out blocks of code in our scripts, we lose that context and the script becomes less readable and harder to troubleshoot.